scripts/release.sh

```diff
@@ -9,13 +9,10 @@ NETWORK=${NETWORK:-"mainnet"}
 
 build() {
     set -e
-    host="$1"
-    name="$2"
-    if [[ "$NETWORK" == "testnet" ]]; then
```

Comment:
so we can rm this [NETWORK](https://github.com/crypto-org-chain/chain-main/blob/0fab1a9847a418a6c4cbcfe570c7589e27b98ab1/scripts/release.sh#L8) right?

Reply:
yes